tests/peerscout/api/main_test.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import json
+
+from flask.testing import FlaskClient
+
+import pytest
+
+from data_science_pipeline.peerscout.api.main import (
+    create_app,
+)
+
+
+LOGGER = logging.getLogger(__name__)
+
+INPUT_DATA_VALID = {
+    "manuscript_id": "12345",
+    "tracking_number": "12345_ABC",
+    "revision_number": 0,
+    "abstract": "<p>Lorem ipsum dolor sit amet, <i>consectetur adipiscing elit</i></p>",
+    "title": "<b>Lorem Ipsum</b>",
+    "author_suggestion": {
+      "include_reviewing_editors_id": [
+        "43038",
+        "178962",
+        "7970"
+      ],
+      "exclude_reviewing_editors_id": [
+        "1000"
+      ],
+      "include_senior_editors_id": [
+        "15332",
+        "42473",
+        "42011"
+      ],
+      "exclude_senior_editors_id": [
+        "73636"
+      ]
+    }
+}
+
+INPUT_DATA_WTIHOUT_ABSTRACT = {**INPUT_DATA_VALID, "abstract": ""}
+
+NO_RECOMENDATION_RESPONSE = {
+    "reviewing_editor_recommendation": {
+        "person_ids": [],
+        "recommendation_html": "<b>No</b> recommendation available"
+    },
+    "senior_editor_recommendation": {
+        "person_ids": [],
+        "recommendation_html": "<b>No</b> recommendation available"
+    }
+}
+
+
+@pytest.fixture(name='test_client')
+def _test_client() -> FlaskClient:
+    app = create_app()
+    return app.test_client()
+
+
+def _get_json(response):
+    return json.loads(response.data.decode('utf-8'))
+
+
+def _get_ok_json(response):
+    assert response.status_code == 200
+    return _get_json(response)
```

Comment:
very minor: looks like we can also use just `.json` (so we may not need the `_get_json` method)